osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/access_denied.html

```diff
@@ -18,7 +18,7 @@
                     {% csrf_token %}
                     <button class="btn btn-primary"><span class="icon default">&#9911;</span> {% trans 'Request access' %}</button>
                 </form>
-                {% blocktrans with next=next_page %}and try <a href="{{ next_page }}">again</a>.{% endblocktrans %}
```

Comment:
It should have, if `next` was defined - but what do we care _now_, because it is being removed :laughing: